res/res_pjsip/pjsip_distributor.c

```diff
@@ -42,7 +42,7 @@ static pjsip_module distributor_mod = {
 struct ast_sched_context *prune_context;
 
 /* From the auth/realm realtime column size */
-#define MAX_REALM_LENGTH 40
+#define MAX_REALM_LENGTH 255
```

Comment:
I know these are duplicated to begin with, but could we have a single definition in a header file, that could then be used in all 3 places, to ensure they all match?

Reply:
Hi InterLiked1,

Thanks, btw, I think I forked from the old one. I will sync the repository and will submit the PR again.